packages/c/css-social-buttons.json

```
@@ -1,14 +1,7 @@
 {
   "name": "css-social-buttons",
   "description": "Zocial CSS social buttons.",
-  "version": "1.3.0",
-  "keywords": [
-    "css",
-    "font",
-    "icon",
-    "social",
-    "zocial"
-  ],
+  "keywords": ["css", "font", "icon", "social", "zocial"],
```

```suggestion
  "keywords": [
    "css",
    "font",
    "icon",
    "social",
    "zocial"
  ],
```
Consistent formatting with other packages.